libnavigation-core/src/main/java/com/mapbox/navigation/core/routeoptions/RouteOptionsUpdater.kt

```diff
@@ -148,9 +147,9 @@ class RouteOptionsUpdater {
             optionsBuilder.arriveBy(null)
             optionsBuilder.departAt(null)
         } catch (e: Exception) {
-            logE(TAG, "routeOptions=[$routeOptions]")
-            logE(TAG, "locationMatcherResult=[$locationMatcherResult]")
-            logE(TAG, "routeProgress=[$routeProgress]")
+            logE("routeOptions=[$routeOptions]")
```

Comment:
As proposed, this log would look like:
```
e/Mapbox: [nav-sdk] routeOptions= RouteOptions{...}
```

This looks a little bit out of context without the information that was previously in the tag. Can we refactor and bake in the existing tags into messages or "categories"?

Reply:
I'm afraid even we might have a hard time understanding these types of generic logs, it could've been logged from anywhere in the SDK without the contextual information.

Reply:
`D/Mapbox: [nav-sdk] [NetworkHandler] NetworkStatus=ReachableViaWiFi` or `D/Mapbox: [NetworkHandler] NetworkStatus=ReachableViaWiFi`. That `[nav-sdk]` is to be consistent with other SDKs.

Discussed internally and `D/Mapbox: [nav-sdk] [NetworkHandler] NetworkStatus=ReachableViaWiFi` should give that extra context.

Let me revert and re-push - that's why I kept the 2 commits 😅